File: backend/semgrep_rules/tests/java/kisa_sw_26_weak_password_policy.java
class kisa_sw_26_weak_password_policy {
    boolean unsafe(String password) {
        // ruleid: kisa.sw26.java.weak-minimum-password-length
        return password.length() >= 6;
    }
    boolean safe(String password) {
        // ok: kisa.sw26.java.weak-minimum-password-length
        return password.length() >= 12;
    }
}

File: backend/semgrep_rules/tests/javascript/kisa_sw_26_weak_password_policy.js
function unsafe(password) {
  // ruleid: kisa.sw26.javascript.weak-minimum-password-length
  return password.length >= 6;
}
function safe(password) {
  // ok: kisa.sw26.javascript.weak-minimum-password-length
  return password.length >= 12;
}
